components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_split_ab2.F

```diff
@@ -1187,8 +1216,16 @@ subroutine ocn_time_integrator_split_ab2(domain, dt)!{{{
 
             ! Compute barotropic thickness at the edge. Note this
             ! matches the sum of baroclinic thicknesses at this edge.
-            call ocn_diagnostic_solve_wctEdge(btrvel_temp, sshSubcycleCur, &
-                      bottomDepthEdge, wctEdge)
+            !$omp parallel
+            !$omp do schedule(runtime)
+            do iCell = 1, nCellsAll
+               deltaSSH(iCell) = sshSubcycleCur(iCell) - sshNew(iCell)
+            end do
+            !$omp end do
+            !$omp end parallel
+
+            call ocn_diagnostic_solve_wctEdge(btrvel_temp, deltaSSH, &
+                      baroclinicThickness, wctEdge)
```

Comment:
@hyungyukang commented in a meeting that this is the critical change in this PR.

Reply:
It looks from https://github.com/E3SM-Project/E3SM/pull/6047 that this would be BFB when the thickness is centered, so I don't think this will affect the instability we encountered in the v3 HR G-case.